business_requirement/models/business.py

```diff
@@ -199,18 +196,23 @@ def _get_default_company(self):
     to_be_reviewed = fields.Boolean(
         string='To be Reviewed'
     )
-    kanban_state = fields.Selection([('normal', 'In Progress'),
-                                     ('on_hold', 'On Hold'),
-                                     ('done', 'Ready for next stage')],
-                                    'Kanban State',
-                                    track_visibility='onchange',
-                                    required=False,
-                                    copy=False, default='normal')
+    kanban_state = fields.Selection(
```

Comment:
@elicoidal @seb-elico Too late for ask use https://github.com/OCA/server-tools/tree/10.0/base_kanban_stage
to add this feature?